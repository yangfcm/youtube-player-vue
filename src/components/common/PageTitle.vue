<template>
  <h2 class="ui red header" style="margin-bottom: 10px;">
    <slot></slot>
  </h2>
</template>

<script>
export default {};
</script>