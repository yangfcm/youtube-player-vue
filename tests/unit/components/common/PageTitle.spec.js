import { shallowMount } from "@vue/test-utils";
import PageTitle from "@/components/common/PageTitle";

test("PageTitle component can render correctly", () => {
  const title = "Title";
  const wrapper = shallowMount(PageTitle, {
    slots: {
      default: title,
    },
  });
  expect(wrapper).toMatchSnapshot();
  expect(wrapper.find("h2").text()).toContain(title);
});
