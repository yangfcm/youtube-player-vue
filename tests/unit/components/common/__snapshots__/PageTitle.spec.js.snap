// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle component can render correctly 1`] = `<h2 class="ui red header" style="margin-bottom: 10px;">Title</h2>`;
